src/v1/sources/shopify/transform.js

```diff
@@ -0,0 +1,314 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const lodash = require('lodash');
+const get = require('get-value');
+const { RedisError } = require('@rudderstack/integrations-lib');
+const stats = require('../../../util/stats');
+const {
+  getShopifyTopic,
+  createPropertiesForEcomEvent,
+  getProductsListFromLineItems,
+  extractEmailFromPayload,
+  getAnonymousIdAndSessionId,
+  checkAndUpdateCartItems,
+  getHashLineItems,
+  getDataFromRedis,
+} = require('./util');
+const logger = require('../../../logger');
+const { RedisDB } = require('../../../util/redis/redisConnector');
+const { removeUndefinedAndNullValues, isDefinedAndNotNull } = require('../../../v0/util');
+const Message = require('../../../v0/sources/message');
+const { EventType } = require('../../../constants');
+const { processEventV2 } = require('./pixelTransform');
+const {
+  INTEGERATION,
+  MAPPING_CATEGORIES,
+  IDENTIFY_TOPICS,
+  ECOM_TOPICS,
+  RUDDER_ECOM_MAP,
+  SUPPORTED_TRACK_EVENTS,
+  SHOPIFY_TRACK_MAP,
+  useRedisDatabase,
+} = require('./config');
+
+const NO_OPERATION_SUCCESS = {
+  outputToSource: {
+    body: Buffer.from('OK').toString('base64'),
+    contentType: 'text/plain',
+  },
+  statusCode: 200,
+};
+
+const identifyPayloadBuilder = (event) => {
+  const message = new Message(INTEGERATION);
+  message.setEventType(EventType.IDENTIFY);
+  message.setPropertiesV2(event, MAPPING_CATEGORIES[EventType.IDENTIFY]);
+  if (event.updated_at) {
+    // converting shopify updated_at timestamp to rudder timestamp format
+    message.setTimestamp(new Date(event.updated_at).toISOString());
+  }
+  return message;
+};
+
+const ecomPayloadBuilder = (event, shopifyTopic) => {
+  const message = new Message(INTEGERATION);
+  message.setEventType(EventType.TRACK);
+  message.setEventName(RUDDER_ECOM_MAP[shopifyTopic]);
```

Comment:
This is old code which is not exactly following the Ecom naming standardisation, also there is some redundancy in the code please clean it up